### ISSUE TITLE
Refine SNAP rules for households with members 60+ and/or with disabilities

### ISSUE THREAD

**Issue by kbottagaro** [Gary-Community-Ventures/benefits-api#556]
Refine SNAP rules for households with members 60+ and/or with disabilities as follows:

- Send medical expenses to PE for households with a member who is 60+ and/or has disabilities. For these HHs, deduct medical expenses above $35. If a HH has medical expenses greater than $35 but less than or equal to $165, use $165 as the standard medical deduction. If medical expenses for these HHs exceed $165, use actual expenses in excess of $35 as the medical deduction. In other words, for these HHs: (1) if medical expenses are $35 or less, do not deduct medical expenses; (2) if medical expenses are greater than $35 and less than or equal to $165, deduct $165 in medical expenses; and (3) if medical expenses are greater than $165, deduct excess over $35. (Regs - Sections 4.407.6, 4.407.61A.; definition "Excess medical deduction")

- For HHs with a member who is 60+ and/or has a disability, do not limit the excess shelter deduction amount. These HHs will get the full excess shelter deduction. (Regs - Section 4.407.3B.)

Source: Colorado regs at https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11590&fileName=10%20CCR%202506-1

**Comment by kbottagaro** [comment]
@sydney-devine @CalebPena The changes in this ticket appear to have been implemented when ticket 566 was completed. However, before I close this ticket, I have a follow-up question for Caleb: 

This sheet shows the profiles I created for SNAP validations - https://docs.google.com/spreadsheets/d/1UEER3_aEvEfFK8anTS1C7iR0c6bq_AFER3MvW_Zq2ik/edit?gid=1774762416#gid=1774762416. My questions relates to the profile in row 8 (for which I haven't yet created a validation). Both SNAP Screener and my manual calculations come in at $271/month, but MFB is showing $275/month. I don't think it's rounding because rounding is usually within $1. This is a HH with a 60+ person and expenses that include $80 in medical expenses (so that in Colorado, there would be a standard medical deduction = $165, unlike NC). Would like Caleb to look at the backend calculation and see where we may differ.
